spec/models/merchant_spec.rb

```diff
@@ -74,4 +74,17 @@
   describe "validations" do
     it { should validate_presence_of(:name) }
   end
+
+  describe 'instance variables' do
+    it 'filters items by status' do
+      merchant_1 = FactoryBot.create(:merchant)
+      item_1 = FactoryBot.create(:item, status: 0, merchant: merchant_1)
```

Comment:
Teach me your ways!  Can't wait to get factorybot into my own tests.  Your's are clean and crisp